ebean-migration-db/src/main/java/io/ebean/migration/db/MigrationPlugin.java

```diff
@@ -0,0 +1,35 @@
+package io.ebean.migration.db;
+
+import io.ebean.migration.MigrationConfig;
+import io.ebean.plugin.Plugin;
+import io.ebean.plugin.SpiServer;
+
+/**
+ * @author Roland Praml, FOCONIS AG
+ */
+public class MigrationPlugin implements Plugin {
+  private MigrationConfig config = new MigrationConfig();
+  private SpiServer server;
+
+  @Override
+  public void configure(SpiServer server) {
+    config.setName(server.name());
+    config.load(server.config().getProperties());
```

Comment:
CHECKME: 

Ebean transfers some values form the DatabaseConfig
```
      final String dbSchema = config.getDbSchema();
      migrationRunner.setDefaultDbSchema(dbSchema); // mig: schema, setCurrentSchema
      Platform platform = config.getDatabasePlatform().platform();
      migrationRunner.setBasePlatform(platform.base().name().toLowerCase());  // mig: basePlatform
      migrationRunner.setPlatform(platform.name().toLowerCase()); // mig: platform or platformName
```
Would mean that some properties have to be migrated